scalableminds/webknossos
# Fix Lookup for Initial Datastore

When inserting initial data on startup, wK checks if there is already a local datastore and tracingstore. If so, none are inserted.
This check was by name (`"localhost"`), which is the default name for these. However, in production this name may differ, causing this check to be falsely negative, resulting in an SQL error. A better way to identify the local datastore in the db is by using its `url` field.
I also did some minor refactoring as suggested by the IDE.

### Steps to test:
- start with fresh schema, local datastore and tracingstore should be inserted
- on second restart, they should not be inserted again
- in the db, change their name to something else
- on another restart, they should still not be inserted again

### Issues:
- fixes #5120

------
